app/pages/sections/order-items/catalogue-solutions/delete/confirmation/contextCreator.js

```diff
@@ -0,0 +1,10 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../../../../config';
+import { addParamsToManifest } from '../../../../../../helpers/contextCreators/addParamsToManifest';
+
+export const getContext = ({ orderId, solutionName }) => {
+  const context = ({
+    ...addParamsToManifest(manifest, { orderId, solutionName })
+  });
+  return context;
+};
```

Comment:
There are some linting issues with this file:

```
app/pages/sections/order-items/catalogue-solutions/delete/confirmation/contextCreator.js     
   2:10  error  'baseUrl' is defined but never used            no-unused-vars
   7:64  error  Missing trailing comma                         comma-dangle
  10:3   error  Newline required at end of file but not found  eol-last
```

Reply:
how are these getting commited to a PR if the linter fails?

Reply:
resolved